src/slic3r/GUI/StatusPanel.cpp

```diff
@@ -1372,14 +1372,14 @@ void StatusPanel::clean_tasklist_info()
 void StatusPanel::create_tasklist_info()
 {
     m_tasklist_caption_sizer = new wxBoxSizer(wxHORIZONTAL);
-    m_text_tasklist_caption  = new wxStaticText(this, wxID_ANY, _L("Printing List"), wxDefaultPosition, wxDefaultSize, 0);
+    m_text_tasklist_caption  = new wxStaticText(m_project_task_panel, wxID_ANY, _L("Printing List"), wxDefaultPosition, wxDefaultSize, 0);
```

Comment:
This is another one I assume is due to wx3.2, when using `this`, I got a warning about it not sizing properly die to the parent.